### ISSUE TITLE
Getting access_denied error when the rule detects that the account is "linkable"

### ISSUE THREAD

**Issue by francocorreasosa** [auth0-extensions/auth0-account-link-extension#6]
While trying to setup the extension for Storezero, I noticed that when I try to sign up with an account that matches the required conditions for linking, the authorization flow breaks with an `access_denied` error and `vAV2y3LvyrZ442fpfOfjkp1Uol6uH4dJ is not defined` as the message.

Webtask rule logs:
```
4:09:13 PM: Connected to storezero-franco
4:09:45 PM: new webtask request 1504120185417.887545
4:09:45 PM: undefined 'Entered Account Link Rule'
4:09:45 PM: finished webtask request 1504120185417.887545 with HTTP 500 in 281ms
4:09:45 PM: setting tenant quarantine
4:09:45 PM: faulting webtask container following request 1504120185417.887545 failure
4:10:27 PM: new webtask request 1504120227896.279945
4:10:28 PM: undefined 'Entered Account Link Rule'
4:10:28 PM: {
  "code": 500,
  "error": "Script generated an unhandled asynchronous exception.",
  "details": "ReferenceError: vAV2y3LvyrZ442fpfOfjkp1Uol6uH4dJ is not defined",
  "name": "ReferenceError",
  "message": "vAV2y3LvyrZ442fpfOfjkp1Uol6uH4dJ is not defined",
  "stack": "ReferenceError: vAV2y3LvyrZ442fpfOfjkp1Uol6uH4dJ is not defined\n    at async.waterfall.callback.user (/data/io/289d43d45c5c4c2fabdb22981c1adeb5/webtask.js:36:17)\n    at fn (/data/sandbox/node_modules/auth0-authz-rules-api/node_modules/async/lib/async.js:638:34)\n    at Immediate._onImmediate (/data/sandbox/node_modules/auth0-authz-rules-api/node_modules/async/lib/async.js:554:34)\n    at processImmediate [as _immediateCallback] (timers.js:396:17)"
}
4:10:28 PM: Code generated an uncaught exception:  ReferenceError: vAV2y3LvyrZ442fpfOfjkp1Uol6uH4dJ is not defined
    at async.waterfall.callback.user (/data/io/289d43d45c5c4c2fabdb22981c1adeb5/webtask.js:36:17)
    at fn (/data/sandbox/node_modules/auth0-authz-rules-api/node_modules/async/lib/async.js:638:34)
    at Immediate._onImmediate (/data/sandbox/node_modules/auth0-authz-rules-api/node_modules/async/lib/async.js:554:34)
    at processImmediate [as _immediateCallback] (timers.js:396:17)
4:10:28 PM: finished webtask request 1504120227896.279945 with HTTP 500 in 238ms
4:10:28 PM: setting tenant quarantine
4:10:28 PM: faulting webtask container following request 1504120227896.279945 failure
4:10:28 PM: new webtask request 1504120228989.519074
4:10:28 PM: error processing webtask request 1504120228989.519074
```

**Comment by francocorreasosa** [comment]
Solved by updating.

**Comment by machuga** [comment]
Just documenting for later - fixed by 433e5bdec516d9c828ba20806a038c5b4c612fc3
